book.rb

```diff
@@ -0,0 +1,20 @@
+class Book
+  attr_accessor :title, :author
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rental = []
+  end
+
+  # book has many rental
+  def add_rental(rental)
+    @rental << rental
+    student.classroom = self
+
+    # a rental belongs to a book
+    # saving the current rental object refered to as self
+    # to the book objects rental variable
+    book.rental = self
```

Comment:
- [ ] Here I think the `add_rental` method should take person and date, and the book itself should be sent as a parameter to create the new rental achieving the 'has-many' association. Something like so: 

```ruby
  def add_rental(person, date)
    Rental.new(date, self, person)
  end
```
Please kindly fix that.